articles/connections/database/custom-db/lock.md

```diff
@@ -0,0 +1,11 @@
+---
+description: Learn how to use Lock with your custom database.
+crews: crew-2
```

Comment:
these crew-2 metadata were never used so we might as well remove them (there is not even a crew-2 in the company any longer). no need to do this as part of this PR but please add this in trello